### PR TITLE
init user list basic feature

### LINKED ISSUE #1: As an admin I want the user id verification can be more agile
currently, we are using hardcode way to detect is the user allowed to upload images or videos,
it needs to be changed.

the solutions may be like:
1. using the existing minion service to create a new bucket to store all allowed user then detect in runtime
2. create a Redis to store all allowed user then detect in runtime

### PR DESCRIPTION
**Pull request by ianchen06**
Added loading userID from a bucket into a map on startup.
On every msg receive, userID is checked if exists in the map before proceeding.

### CODE REVIEW

File: main.go [hunk 1]
```diff
@@ -68,6 +70,15 @@ func main() {
 		log.Fatal(err)
 	}
 
+	// Create a done channel.
+
+	doneCh := make(chan struct{})
+	defer close(doneCh)
+	userList = make(map[string]struct{})
+	for userObj := range mc.ListObjects(userBucket, "", true, doneCh) {
```
Comment (tommady) [inline main.go:78]: is the list objects function will keep going to put objects to userObj chan when there has a new item until the doneCh closed?
please dig into see how it implemented. :bow

File: main.go [hunk 2]
```diff
@@ -97,7 +108,8 @@ func callbackHandler(w http.ResponseWriter, r *http.Request) {
 				return nil
 			}
 
-			if event.Source.UserID != "U00d6815afb6f57eb9ac959614d2520db" && event.Source.UserID != "U5b9f3b6e6636930d434372d70ce9c9b0" {
+			_, ok := userList[event.Source.UserID]
+			if ok == false {
```
Comment (tommady) [inline main.go:112]: we can combine 111 and 112 to 
`
if _, exist := userList[event.Source.UserID]; !exist {
        return nil
}
`

File: main.go (hunk 1 above)
Comment (ianchen06) [reply main.go:78]: @tommady 
Thanks for your review.
https://github.com/minio/minio-go/blob/79f99bc54ad036fd2ef9166149e9a8de33920edc/api-list.go#L431
The goroutine will end after the result is received.

Comment (tommady) [reply main.go:78]: cool awesome, then we need to find a way to trigger reload user bucket with multiple pods.

**Pull request merged**